test/retrieve_request_handler_tests.exs

```diff
@@ -0,0 +1,17 @@
+defmodule HTTParrot.RetrieveRequestHandlerTests do
+  use ExUnit.Case
+  import :meck
+  import HTTParrot.RetrieveRequestHandler
+
+  setup do
+    HTTParrot.RequestStore.clear(:test)
```

Comment:
Because we are using meck, you need to add this to the setup:

`on_exit fn -> unload end`